lib/class-wp-rest-autosaves-controller.php

```diff
@@ -0,0 +1,290 @@
+<?php
+/**
+ * REST API: WP_REST_Autosaves_Controller class.
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since 5.0.0
```

Comment:
We were using the Gutenberg version instead elsewhere, maybe it's time to switch to the WP version?

Reply:
yea, i wrote this thinking about it getting merged into core separately; please adjust if versioning isn't correct.